ansible/roles/ipaclient/README.md

```diff
@@ -0,0 +1,209 @@
+ipaclient role
```

Comment:
Where does this role come from ?

Can't you add it to galaxy requirements ?

See https://github.com/redhat-cop/agnosticd/blob/development/docs/How_to_use_galaxy_roles_in_your_config.adoc

Reply:
It comes from [ansible-freeipa](https://github.com/freeipa/ansible-freeipa). It is available from Ansible Galaxy.